test/common/test_helpers.h

```diff
@@ -14,6 +14,12 @@
 extern "C" {
 #endif
 
+int is_filled_with_char(void *ptr, size_t size, char c);
```

Comment:
could you use CamelCase style to match naming convention already used in this file? consider "bufferIsFilledWithChar" etc. Also we could keep the declaration of this group of functions together, without an extra empty line between them

Reply:
But it is a *.c file, so it should not use CamelCase, so I would prefer to change the rest to not use the CamelCase in a separate PR.

Reply:
ok lest unify this in the separate PR

Reply:
Done: https://github.com/oneapi-src/unified-memory-framework/pull/76